x/evm/keeper/deferred.go

```diff
@@ -63,3 +63,15 @@ func (k *Keeper) AppendToEvmTxDeferredInfo(ctx sdk.Context, bloom ethtypes.Bloom
 	}
 	prefix.NewStore(ctx.TransientStore(k.transientStoreKey), types.DeferredInfoPrefix).Set(key, bz)
 }
+
+func (k *Keeper) GetEVMTxDeferredInfo(ctx sdk.Context) *types.DeferredInfo {
```

Comment:
side thought: It can sometimes be nice sdk-wise to have signatures that return `*pointer, bool` where the bool represents `found`